_data/reference.yml

```diff
@@ -1,41 +1,62 @@
 bigheader: "Reference Documentation"
 abstract: "Design docs, concept definitions, and references for APIs and CLIs."
 toc:
-- docs/reference.md
+- docs/reference/index.md
 
-- title: "Kubernetes Resource Types"
+- title: "API Reference"
   section:
   - title: Version 1.6
-    path: /docs/resources-reference/v1.6/
+    section:
+    - title: API
```

Comment:
I think it should go like this:

- API Reference
   - v1.6 API
   - v1.5 API